spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -411,6 +411,7 @@
       context 'but doc_pii validation fails' do
         let(:first_name) { 'FAKEY' }
         let(:last_name) { 'MCFAKERSON' }
+        let(:address1) {'123 Houston Ave'}
```

Comment:
@matthinz Requesting your review specifically because I'd like extra eyes on these changes in `spec/controllers/idv/image_uploads_controller_spec.rb`. Do these look reasonable to you?